sjb/config/common/test_cases/origin_built_installed_release.yml

```diff
@@ -105,7 +107,10 @@ extensions:
                          -e etcd_data_dir="${ETCD_DATA_DIR}" \
                          -e openshift_master_default_subdomain="${local_ip}.nip.io"             \
                          -e openshift_pkg_version="$( cat ./ORIGIN_PKG_VERSION )"               \
+                         -e openshift_release="$( cat ./ORIGIN_RELEASE )"                       \
```

Comment:
Setting openshift_release with extra_vars is bad.

We should be removing extra_vars usage, not expanding it.

Reply:
This is how the current Origin jobs do it -- what is better? Can you post a snippet of how you'd rather have this info passed in?

Reply:
I don't have strong feelings about this, but our jobs are wildly inconsistent about this right now and I would love for them not to be.

Reply:
export these values as environment variables, then use lookups:

https://github.com/openshift/openshift-ansible/blob/master/test/ci/inventory/group_vars/all.yml#L13

Reply:
How are we pulling in that file _right now_?

Reply:
Even if we should stop using extra_vars, this seems like a step in the right direction by consolidating them all in one place, or at the very least fewer places.